src/components/ToolTip/ToolTip.less

```diff
@@ -0,0 +1,323 @@
+@tooltip-pointer-inner-width: 8px;
```

Comment:
Should be `@ToolTip-pointer-inner-width: 8px;` per [contrib](https://github.com/appnexus/lucid/blame/master/CONTRIBUTING.md#L60)